src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Simplification/Simplifiers/CastSimplifier.cs

```diff
@@ -461,7 +461,11 @@ private static bool CastRemovalCouldCauseSignExtensionWarning(ExpressionSyntax c
                 return true;
 
             // We can safely remove convertion to object in interpolated strings regardless of nullability
-            if (castNode.IsParentKind(SyntaxKind.Interpolation) && originalConversionOperation.Type?.SpecialType is SpecialType.System_Object)
+            if (castNode.IsParentKind(SyntaxKind.Interpolation) && originalConvertedType.SpecialType is SpecialType.System_Object)
+                return true;
+
+            // Can remove convertion of string literal to string regardless of nullability
```

Comment:
Convertion

Reply:
What is the issue here? I don't see anything misspelled

Reply:
Should be "conversion" I think.

Reply:
Uh, yes, thanks @Youssef1313!